README.md

```diff
@@ -37,6 +37,14 @@ Whenever you want to update:
 cargo mobile update
 ```
 
+### Windows
+
+We need to tell `cargo` to not install the `cargo-apple` binary on Windows.
```

Comment:
There's a solution for this in #5 that I'm happy with, so you can just rebase onto that once it's merged.

Reply:
o great, this was bugging me